circle_suck/base/constants.py

```diff
@@ -1,5 +1,16 @@
 from base.utils import merge_dicts
 
+# list of sports as choices.
+SPORTS_CHOICES = [
+    ('Football', 'Football')
```

Comment:
Instead of being redundant, just list them out as `['Football', 'Volleyball', etc]` and then in the model, do `[(x, x) for x in SPORTS]`